src/command/marian_scorer.cpp

```diff
@@ -2,13 +2,14 @@
 
 #include "models/model_task.h"
 #include "rescorer/rescorer.h"
+#include "common/timer.h"
```

Comment:
Inspired by missing autoformatting: what do you think about running a small subset of rules from the code autoformatter in a regular basis, for example, weekly? Only some mechanical and  usually very safe rules, such as trailing whitespaces, needless new lines, tabs instead spaces, unsorted includes, spaces after keywords, etc. Jenkins could open pull requests with those for us.

Reply:
Maybe put this question into an issue?